CourseGrab/Extensions/UIFont+Shared.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIFont+Shared.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 1/26/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+extension UIFont {
+
+    static let _14RegularFont = UIFont.systemFont(ofSize: 14)
+    static let _16RegularFont = UIFont.systemFont(ofSize: 16)
+    static let _18RegularFont = UIFont.systemFont(ofSize: 18)
+
+    static let _14MediumFont = UIFont.systemFont(ofSize: 14, weight: .medium)
+
+    static let _12SemiboldFont = UIFont.systemFont(ofSize: 12, weight: .semibold)
+    static let _16SemiboldFont = UIFont.systemFont(ofSize: 16, weight: .semibold)
+    static let _18SemiboldFont = UIFont.systemFont(ofSize: 18, weight: .semibold)
+    static let _20SemiboldFont = UIFont.systemFont(ofSize: 20, weight: .semibold)
+
+    static let _20BoldFont = UIFont.systemFont(ofSize: 20, weight: .bold)
+    static let _30BoldFont = UIFont.systemFont(ofSize: 30, weight: .bold)
+    static let _36BoldFont = UIFont.systemFont(ofSize: 36, weight: .bold)
```

Comment:
Maybe we could get rid of `Font` in every variable name (i.e. `_20Bold`, etc.). It is clearly a font because it is `UIFont._20Bold` and its type is UIFont?